code/modules/mob/living/carbon/human/species/station/prometheans.dm

```diff
@@ -170,15 +170,40 @@ var/datum/species/shapeshifter/promethean/prometheans
 		H.adjustToxLoss(3 * heal_rate)	// Tripled because 0.5 is miniscule, and fire_stacks are capped in both directions
 		healing = FALSE
 
+	//Prometheans automatically clean every surface they're in contact with every life tick - this includes the floor without shoes.
+	//They gain nutrition from doing this.
 	var/turf/T = get_turf(H)
 	if(istype(T))
-		var/obj/effect/decal/cleanable/C = locate() in T
-		if(C && !(H.shoes || (H.wear_suit && (H.wear_suit.body_parts_covered & FEET))))
-			qdel(C)
```

Comment:
This specifically was present so that they would actually remove dirt effects, blood, vomit, etc, when cleaning a tile, rather than leaving them.

Reply:
If I am reading it right, those effects delete src when `clean_blood` is run, since space cleaner doesn't call them out either.

Reply:
Ahhh, alright